bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -114,6 +114,11 @@ spec:
             exec:
               command: ["/usr/share/openvswitch/scripts/ovs-ctl", "stop"]
         terminationGracePeriodSeconds: 10
+        env:
+        - name: OVN_EXTENSION_ENABLE
```

Comment:
@JacobTanenbaum let's change these to OVN_HYBRID_OVERLAY_ENABLE and OVN_HYBRID_OVERLAY_NET_CIDR